apiserver/params/crossmodel.go

```diff
@@ -299,6 +300,23 @@ type GetTokenArg struct {
 	Tag string `json:"tag"`
 }
 
+// ImportEntityArgs holds the arguments to an ImportRemoteEntity API call.
+type ImportEntityArgs struct {
+	Args []ImportEntityArg
+}
```

Comment:
This is required by the rpc infrastructure juju uses. There needs to be a containing struct